_ml-commons-plugin/remote-models/index.md

```diff
@@ -205,7 +205,18 @@ Take note of the returned `model_id` because you’ll need it to deploy the mode
 
 ## Step 4: Deploy the model
 
-To deploy the registered model, provide its model ID from step 3 in the following request:
+From 2.13, we support automatically deploy remote model by default so this step can be skipped. You can disable it by setting plugins.ml_commons.model_auto_deploy.enable as false. To undeploy the model, please use the undeploy API to undeploy it.
```

Comment:
Should we remove this step altogether and just add a sentence to the previous step: `Starting with OpenSearch version 2.13, externally hosted models are deployed automatically by default when you send a Predict API request for the first time.`

Reply:
Let's keep this step because we still support manual deploy for the sake of BWC.